data/assets/examples/renderable/renderablepointcloud/pointcloud_textured_facecameraposition.asset

```diff
@@ -0,0 +1,37 @@
+-- Orientation - Face Camera Position
+-- This example creates a point cloud where the planes that make up the points are
+-- oriented to face the camera position, rather than the view direction (which is
+-- default). This means that the point will be layed out in a spherical way around the
+-- camera, which works better for spherical displays compared to the default
+-- orientation.
+--
+-- A texture is added so that we can more easily see how the orientation is altered.
+-- See Textured example for more details.
+
+local Node = {
+  Identifier = "RenderablePointCloud_Example",
```

Comment:
Won't this identifier collide with other examples if I were to load several of them at the same time?

Reply:
Yes!  I have forgotten to change it form the example I copy-pasted ;)  Thank you